test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -202,6 +202,12 @@ def test_r2r(self, case, gop, slices, bitrate, fps, quality, refs, profile):
     vars(self).setdefault("r2r", 5)
     self.encode()
 
+  @slash.parametrize(*gen_hevc_vbr_lp_parameters(spec, ['main']))
```

Comment:
We probably want to create a new, TCBRC variant specific case generator for this.